standard/expressions.md

```diff
@@ -3852,6 +3890,7 @@ For an operation of the form `x == y` or `x != y`, if any applicable `operat
 <!-- markdownlint-enable MD028 -->
 > *Example*: The example
 >
+> <!-- Example: {template:"standalone-console", name:"ReferenceTypeEqualityOperators2", expectedOutput:["True","False","False","False"]} -->
```

Comment:
Note to self: it would be lovely to be able to say "derive the expected output from the subsequent console output section of the spec". Once all the examples are *working*, we can see whether we're consistently using the same fencing etc.